### PR TITLE
Response in json

### PR DESCRIPTION
**Pull request by Tarun-Rajj**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "input": "Hii Sir",
+    "Python": "Developer"
+}
```
Comment (ghost) [inline data.json:5]: No need to push json file

File: app.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#         'description': 'Take_Input, Title, description,',
+#         'done': False
+#     },
+#     {
+#         'id': 2,
+#         'title': 'Learn Python',
+#         'description': 'Need to find a good Python tutorial on the web',
+#         'done': False
+#     }
+# ]
```
Comment (ghost) [inline app.py]: remove these comments

File: app.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                     return jsonify({"Error": str(e)}), 500
+                
+#Route to accept pull requests
+
+@app.route("/put", methods=['PUT'])
+def update_data():
+    try:
+        data = request.get_json()
+        if 'input' in data:
+            with open("data.json","w") as json_file:
```
Comment (aayush-kumar-codes) [inline app.py:43]: Will this update the file or overwrite it?

File: app.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        data = request.get_json()
+        if 'input' in data:
+            with open("data.json","r+") as json_file:
+                content = json.load(json_file)
+                content["input"] = data["input"]
+
+            #update the data in json file
+            with open("data.json","w") as json_file:
+                json.dump(content,json_file)
+            return jsonify({"message":"Data patched successfully."})
```
Comment (aayush-kumar-codes) [inline app.py:67]: Status should be 204

File: app.py [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+@app.route('/post',methods=['POST']) #This line defines a route for handling GET requests.
+def store_input():
+            try:
+                data = request.get_json()
+                print(data)
+                if data:
+                    with open('data.json', 'w') as json_file:
+                        json_file.write(json.dumps(data, indent=4))
+                    return jsonify({"message": "Data stored successfully"})
```
Comment (aayush-kumar-codes) [inline app.py:17]: Status should be 201

**Pull request merged**